src/main.c

```diff
@@ -446,8 +453,24 @@ static bool hint(void)
 	return false;
 }
 
+/* function to compose message string with elapsed time */
+static void elapsed_time_str(const char *msg, char *time_string)
+{
+	int hours, minutes, seconds;	//variables to compose time string hh:mm:ss
+
+	if(g_playing)			//only if playing
```

Comment:
this comment is obvious. Generally there are too many comments. Just comment some details that are not easily understood by the code itself.

Reply:
I haven't been producing software for many years now, now it's just for hobby and fun, I like to comment to be useful to those who are new to studying the C language, for them it's important to understand even trivial things, for experts it's superfluous for sure...